src/openforms/contrib/kvk/tests/test_validators.py

```diff
@@ -53,6 +55,11 @@ def test_invalid_branchNumber(self):
 class KvKRemoteValidatorTestCase(OFVCRMixin, KVKTestMixin, SimpleTestCase):
     VCR_TEST_FILES = TEST_FILES
 
+    def setUp(self) -> None:
+        super().setUp()
+
+        clear_caches()
```

Comment:
```suggestion
    def setUp(self) -> None:
        super().setUp()

        clear_caches()
        self.addCleanup(clear_caches)
```
this ensures that after a test, the cache is cleared too